### PR TITLE
Create Facet filters.md

### PR DESCRIPTION
**Pull request by abankalarm**
facet filters documentation

### CODE REVIEW

**Comment by abankalarm** [comment]
@MarinPostma hey can you review this again changes suggested have been done

File: guides/advanced_guides/Facet filters.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Errors are sent to the user on:
+  1. Incorrect syntax usage`[["color:red"]]` and `["color:red"]` are examples correct syntax while `"color:red"` is incorrect.
+  2. Faceting on existing facets. It is asked of the user to do faceting on an unregistered or not existing facet. (we could send the list of existing facets to the user).
+    
+The facets accepts a `[String]` containing all the facets for which we want to retrieve count information. It defaults to `Null`. If it is set to `["*"]`, counts for all facets set above will be returned. It will return facets based on a best effort to match the user request. If a facet name is not existing it will be ignored.
+    
+## Additional fields
+The result of the search query contains two additional fields if the `facets` parameter has been set:
+  1. `Facets`: returning a count of candidates for each requested facets in the facets parameter, if set to `[*]` a count for all facets is returned.
+  2. `ExhaustiveFacetsCount` telling whether the above count is exhaustive or approximative.
```
Comment (MarinPostma) [inline guides/advanced_guides/Facet filters.md:34]: This example should be like the one we already have on the documentation.

use this [one](https://docs.meilisearch.com/guides/advanced_guides/filtering.html#examples) as refererence

Comment (MarinPostma) [inline guides/advanced_guides/Facet filters.md:34]: this goes in the search section above. Try to distinguish between what must be sent, and what is returned from the server.

File: guides/advanced_guides/Facet filters.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# INTRO
+
+While filtering output has been a general approach, facets differ from basic filtering as they are aimed at restricting the document to be filtered on the up stream. To achieve this, facets are declared at indexing. In other words it can be said that, Facets are document attributes that indicate which category a document belongs to. The ultimate goal of faceting is to be complimentary with filtering.
+
```
[the remark this answers is not in the document]
Comment (MarinPostma) [reply guides/advanced_guides/Facet filters.md:4]: after a light intro on faceted search, I would even have a section explaining in depth how it works.

**Comment by bidoubiwa** [comment]
This PR is innactive. Closing.

**Pull request closed without merging**